orc8r/cloud/go/services/certifier/tokens.go

```diff
@@ -0,0 +1,102 @@
+package certifier
+
+import (
+	"crypto/rand"
+	"errors"
+	"fmt"
+	"hash/crc32"
+	"strings"
+
+	"github.com/jxskiss/base62"
+)
+
+type TokenType string
+
+const (
+	Personal TokenType = "op"
+)
+
+// Length of checksum of token's byte array
+const checksumLength = 4
+
+// GenerateToken generates a random 32-byte token with a checksum
+func GenerateToken(tokenType TokenType) (string, error) {
```

Comment:
Nit: any time you see a "stutter", it's a good place to be more concise

```suggestion
func GenerateToken(typ TokenType) (string, error) {
```